apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/create-note-about-photograph.e2e.cy.ts

```diff
@@ -0,0 +1,116 @@
+describe(`photograph detail view`, () => {
+    describe(`note creation`, () => {
+        const photographId = `9b1deb4d-3b7d-4bad-9bdd-2b0d7b110002`;
+
+        const createNoteLabel = 'Create Note';
+
+        beforeEach(() => {
+            cy.visit(`/Resources/Terms/${photographId}`);
+        });
+
+        describe(`when the user is not logged in`, () => {
+            it(`should not be available`, () => {
+                cy.contains(createNoteLabel).should('not.exist');
+            });
+        });
+        //  TODO Add aditional non-admin test users and test this
+        // describe(`when the user is logged-in, but not an admin`, () => {
+        //     it(`should not display the panel`, () => {
+        //         cy.get('woo hooo hooo not found~ test me!');
+        //     });
+        // });
+
+        describe(`when the user is logged in as COSCRAD admin`, () => {
+            beforeEach(() => {
+                cy.login();
+
+                cy.getByDataAttribute('nav-menu-icon').click();
+
+                cy.get('[href="/Resources"] > .MuiButtonBase-root').click();
+
+                cy.getByDataAttribute('photograph').click();
```

Comment:
use `cy.navigateToIndex('photograph')`.

Note we probably want to include the next cy.get in part of a `cy.navigateToDetail('photograph')`, but that's not there yet.